code/game/objects/items/miscellaneous.dm

```diff
@@ -145,3 +145,60 @@
 	user.gib()
 	playsound(src, 'sound/items/eatfood.ogg', 50, TRUE, -1)
 	return MANUAL_SUICIDE
+
+/obj/item/virgin_mary
+	name = "A picture of the virgin mary"
+	desc = "A small, cheap icon depicting the virgin mother."
+	icon = 'icons/obj/items_and_weapons.dmi'
+	icon_state = "virgin_mary"
+	resistance_flags = FLAMMABLE
+	///Has this item been used already.
+	var/used_up = FALSE
+	///List of mobs that have already been mobbed.
+	var/static/list/mob_mobs = list()
+
+
+/obj/item/virgin_mary/attackby(obj/item/W, mob/user, params)
+	var/ignition_msg = W.ignition_effect(src, user)
+	if(ignition_msg)
+		if(resistance_flags & ON_FIRE)
+			return
+		user.dropItemToGround(src)
+		user.visible_message("<span class='danger'>[user] lights [src] ablaze with [W]!</span>", "<span class='danger'>You light [src] on fire!</span>")
+		fire_act()
+		if(used_up)
+			return ..()
+		if(!isliving(user) || !user.mind) //A sentient mob needs to be burning it, ya cheezit.
+			return ..()
+		var/mob/living/joe = user
+
+		if(joe in mob_mobs) //Only one nickname fuckhead
+			to_chat(joe, "<span class='warning'>You have already been initiated into the mafioso life.</span>")
+			return FALSE
+
+		to_chat(joe, "<span class='notice'>As you burn the picture you feel like you can pick up a fitting nickname</span>")
```

Comment:
. at the end.

Reply:
and maybe a comma in the middle or something
"As you burn the picture you feel like you can pick up a fitting nickname" is really awkward